src/geocat/f2py/triple_to_grid_wrapper.py

```diff
@@ -361,10 +361,10 @@ def triple_to_grid(data,
 
         data = xr.DataArray(
             data.data,
-            # coords={
-            #     data.dims[-1]: x_in,
-            #     data.dims[-2]: y_in,
-            # },
+        # coords={
+        #     data.dims[-1]: x_in,
+        #     data.dims[-2]: y_in,
+        # },
```

Comment:
another linter modification I don't agree with.
Why is Yapf behaving differently with this implementation? it should still be using the google style that we already had right?

Reply:
See above resolution.

Reply:
resolved